src/components/SocialMediaIcons/SocialMediaIcons.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```

Comment:
This component already exists in `src/components/SocialMediaIconsComponent` named `AnimatedIcons.jsx` so we can use that only instead.